### PR TITLE
Begin Docs overhaul

### PR DESCRIPTION
**Pull request by hollyos**
In this PR:
- Update the content for the introduction page
    - add links to docs content
    - update about copy
    - section out content for relevance
- adjust the quick start and separate out packages content to its own page
    - packages content is merged to one page with sections for Docker and Files packages (existing content from "Quick Start")
    - quick start files is removed and merged into the packages content
- Add EOL for v1 API
- Updating overall theming to align with Scarf color scheme
- Add Scarf favicon
- Add registered logo to header
- remove "auto" theming - is only available in premium mkdocs - will now only toggle between light and dark

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
 # Introduction
 
-Scarf is a platform for download and adoption metrics for open source software projects. 
+Welcome to Scarf's documentation, your one-stop guide for understanding and leveraging our platform's powerful tools to track metrics for your open-source software projects.
```
Comment (aviaviavi) [inline docs/index.md:3]:
```suggestion
Welcome to Scarf's documentation, your one-stop guide for understanding and leveraging our platform's powerful tools to track usage metrics for your open-source software projects.
```

File: mkdocs.yml [hunk 2]
```diff
@@ -23,18 +26,25 @@ theme:
...
       toggle:
         icon: material/brightness-4
-        name: Switch to system preference
-site_description: |
-  Scarf's Technical Documentation & API Reference
+        name: Switch to light mode
+  font:
+    text: Roboto
+    code: Roboto Mono
+  favicon: assets/new_scarf.svg
```
Comment (aviaviavi) [inline mkdocs.yml:33]: I think we may want to pick a different color for the favicon, at least for me it's either hard to see or fully invisible when the tab is currently selected:

Screenshot 2023-07-31 at 1 46 30 PM
Screenshot 2023-07-31 at 1 46 24 PM

Maybe we should do blue? Or black? Or non-transparent background? I don't feel strongly.

**Review by aviaviavi — COMMENTED** [review]
Great overhaul! A few small suggestions and one note before merging but once those are addressed looks good to me.

File: mkdocs.yml (hunk 2 above)
Comment (hollyos) [reply mkdocs.yml]: Switched this to the blue icon
icon

File: docs/data-export.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ To export data out of Scarf, go to the main dashboard and click "Export packages
 
 ![Export packages data](assets/pics/data-export/export-packages-data.png)
 
-The data you can export from Scarf includes all events (defined as package downloads and pixel views) from every user that has interacted with your Scarf-enabled artifacts (packages and pixels). Upon clicking "Export packages data", this data will download as a .csv file. 
+The data you can export from Scarf includes all events (defined as package downloads and pixel views) from every user that has interacted with your Scarf-enabled artifacts (packages and pixels). Upon clicking "Export packages data", this data will download as a .csv file.
 
-You can also export this data using [the Scarf API](https://docs.scarf.sh/api-v2/).
+You can also export this data using [the Scarf API](/api-v2).
```
Comment (fabioluz) [inline docs/data-export.md:11]: Is this link correct? We are now using api-docs.scarf.sh

Comment (hollyos) [reply docs/data-export.md:11]: not anymore... I'll need to update to the new link

**Review by aviaviavi — APPROVED** [review]
Once we fix the API link fabio pointed out, :shipit:

File: docs/data-export.md (hunk 3 above)
Comment (aviaviavi) [reply docs/data-export.md:11]: Ping me once that update is looking good and I'll merge!

**Pull request merged**